demos/selective_dualarm_grasping/CMakeLists.txt

```diff
@@ -0,0 +1,205 @@
+find_package(catkin REQUIRED
+    dynamic_reconfigure
+    jsk_recognition_msgs
+    message_generation
+    roseus
+    std_msgs
+)
+
+## System dependencies are found with CMake's conventions
+# find_package(Boost REQUIRED COMPONENTS system)
+
+
+## Uncomment this if the package has a setup.py. This macro ensures
+## modules and global scripts declared therein get installed
+## See http://ros.org/doc/api/catkin/html/user_guide/setup_dot_py.html
+catkin_python_setup()
+
+################################################
+## Declare ROS messages, services and actions ##
+################################################
+
+## To declare and build messages, services or actions from within this
+## package, follow these steps:
+## * Let MSG_DEP_SET be the set of packages whose message types you use in
+##   your messages/services/actions (e.g. std_msgs, actionlib_msgs, ...).
+## * In the file package.xml:
+##   * add a build_depend tag for "message_generation"
+##   * add a build_depend and a run_depend tag for each package in MSG_DEP_SET
+##   * If MSG_DEP_SET isn't empty the following dependency has been pulled in
+##     but can be declared for certainty nonetheless:
+##     * add a run_depend tag for "message_runtime"
+## * In this file (CMakeLists.txt):
+##   * add "message_generation" and every package in MSG_DEP_SET to
+##     find_package(catkin REQUIRED COMPONENTS ...)
+##   * add "message_runtime" and every package in MSG_DEP_SET to
+##     catkin_package(CATKIN_DEPENDS ...)
+##   * uncomment the add_*_files sections below as needed
+##     and list every .msg/.srv/.action file to be processed
+##   * uncomment the generate_messages entry below
+##   * add every package in MSG_DEP_SET to generate_messages(DEPENDENCIES ...)
+
+## Generate messages in the 'msg' folder
+add_message_files(
+  FILES
+  GraspClassificationResult.msg
+)
+
+## Generate services in the 'srv' folder
+add_service_files(
+  FILES
+  GetAnother.srv
+)
+
+## Generate actions in the 'action' folder
+# add_action_files(
+#   FILES
+#   Action1.action
+#   Action2.action
+# )
+
+## Generate added messages and services with any dependencies listed here
+generate_messages(
+  DEPENDENCIES
+  std_msgs
+  jsk_recognition_msgs
+)
+
+################################################
+## Declare ROS dynamic reconfigure parameters ##
+################################################
+
+## To declare and build dynamic reconfigure parameters within this
+## package, follow these steps:
+## * In the file package.xml:
+##   * add a build_depend and a run_depend tag for "dynamic_reconfigure"
+## * In this file (CMakeLists.txt):
+##   * add "dynamic_reconfigure" to
+##     find_package(catkin REQUIRED COMPONENTS ...)
+##   * uncomment the "generate_dynamic_reconfigure_options" section below
+##     and list every .cfg file to be processed
+
+add_custom_target(${PROJECT_NAME}_install_models ALL COMMAND ${PROJECT_SOURCE_DIR}/scripts/install_models.py)
```

Comment:
Could you imitate https://github.com/start-jsk/jsk_apc/commit/40c5cea0a9501941213a5304194ae8a3fef0f030 for the following two reasons?
- To avoid downloading heavy data (each model is about 500MB) from GoogleDrive on build time because it easily reaches to access limit and travis fails
- To avoid placing heavy data in `~/.ros/data` directory on travis machine because that directory must be able to be unpacked from cache before travis timeout (10 min)
  - [That directory is packed and uploaded as cache](https://github.com/start-jsk/jsk_apc/blob/ff56ae305b2439f42d03a0480715b639065cd840/.travis.yml#L15) when travis succeeds
  - In the next travis test, travis machine downloads this cache and tries to unpack it, but current unpacking already seems to take around 10 min. Adding over 1GB data probably causes timeout in unpacking (see https://github.com/start-jsk/jsk_apc/pull/2707#issuecomment-627229028 and https://github.com/start-jsk/jsk_apc/pull/2712)

Reply:
I stop downloading models during catkin build